UI/Selenium/Steps/CheckAlertsCommandCenterSteps.cs

```diff
@@ -289,13 +276,12 @@ public void ThenTheTheVideoHearingsOfficerSeeTheAlertFailedSelf_TestNoToCameraPa
         [Then(@"the the Video Hearings Officer see the alert Failed self-test \(No to Microphone\) participant F & L name")]
         public void ThenTheTheVideoHearingsOfficerSeeTheAlertFailedSelf_TestNoToMicrophoneParticipantFLName()
         {
+            Driver = GetDriver(_hearing.Participant.Where(a => a.Role.Name.ToLower() == "vho").FirstOrDefault().Id, _scenarioContext);
             _hearing = (Hearing)_scenarioContext["Hearing"];
             var alerts = Driver.FindElements(SelectYourHearingListPage.FailedAlert);
-            Assert.IsTrue(alerts.Count > 0);
-
+            alerts.Count.Should().BeGreaterThan(0);
```

Comment:
Same again, can there be 1 or many alerts?

Reply:
See above